plugins/source-terasology-engine/gatsby-node.js

```diff
@@ -0,0 +1,103 @@
+const { graphql } = require("@octokit/graphql");
+const { DateTime } = require("luxon");
+const fs = require("fs");
+
+const PLUGIN_NAME = "source-terasology-engine";
+
+const query = `
+query Engine {
+  organization(login: "MovingBlocks") {
+    repository(name: "Terasology") {
+      issues(first: 10, filterBy: {labels: "Good First Issue", states: OPEN}, orderBy: {field: UPDATED_AT, direction: DESC}) {
+        nodes {
+          id
+          title
+          author {
+            login
+          }
+          labels(first: 10) {
+            nodes {
+              name
+            }
+          }
+          updatedAt
+          url
+        }
+      }
+    }
+  }
+}
+`;
+
+exports.onPreInit = ({ reporter }) =>
+  reporter.verbose("Loaded source-terasology-engine");
+
+exports.sourceNodes = async (
+  {
+    actions: { createNode },
+    createContentDigest,
+    createNodeId,
+    reporter,
+    cache,
+  },
+  { accessToken }
+) => {
+  const gql = graphql.defaults({
+    headers: {
+      authorization: `token ${accessToken}`,
+    },
+  });
+
+  const lastFetchedKey = "terasology-engine-last-fetched";
+  const dataKey = "terasology-engine-data";
```

Comment:
I think the caches are scoped for the plugins individually, so prefixing is not strictly required. This applies to the other plugin as well, so that's rather a note to self here...

Reply:
guess if we plan to merge the plugins anyway, we can make use of the prefix for scoping then, so keeping it for now.